src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml

```diff
@@ -33,17 +37,39 @@ jobs:
       ${{ parameters.poolInternal }}
   strategy:
     matrix:
-      Fedora33-Online:
+      CentOS7-Online:
+        _BootstrapPrep: true
         _BuildArch: x64
-        _Container: ${{ parameters.fedora33Container }}
+        _Container: ${{ parameters.centOS7Container }}
+        _ExcludeOmniSharpTests: true
         _RunOnline: true
+      CentOS7-Offline:
```

Comment:
I don't think this was merged correctly.  CentOS7-Offline should be moved into the `${{ if ne(variables['Build.Reason'], 'PullRequest') }}:` body.  I will push a fix for this.

Reply:
Fixed